plugins/premium/tinymcespellchecker.md

```diff
@@ -145,6 +145,29 @@ tinymce.init({
 });
 ```
 
+{% include configuration/content-langs.md %}
+
+### `spellchecker_select_languages`
```

Comment:
This shouldn't have been moved, as settings are normally in alphabetical order (some older pages aren't unfortunately). So please move this back to it's original location.